CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
```

Comment:
Totes optional: in case you wanna get this out quickly, feel free to do the 8.0.2 release in this PR (entirely up to you though, just mentioning in case it saves you time/hassle of getting another review for essentially the same change)